src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/decisionmaker/deciders/QueueHealthDecider.java

```diff
@@ -36,14 +36,13 @@
   public static final String NAME = "queue_health";
 
   private QueueRejectionClusterRca queueRejectionRca;
-  List<String> actionsByUserPriority = new ArrayList<>();
+  List<String> actionsByUserPriority = null;
   private int counter = 0;
 
   public QueueHealthDecider(long evalIntervalSeconds, int decisionFrequency, QueueRejectionClusterRca queueRejectionClusterRca) {
     // TODO: Also consume NodeConfigurationRca
     super(evalIntervalSeconds, decisionFrequency);
     this.queueRejectionRca = queueRejectionClusterRca;
-    configureActionPriority();
```

Comment:
Shouldn't this be replaced by a call to configureActionPriority(DeciderActionPriorityReader.getActionPriorityOrder(name()))?